.github/workflows/tests.yml

```diff
@@ -12,7 +12,10 @@ jobs:
       max-parallel: 3
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: [3.7, 3.8]
+        python-version: [3.7, 3.10.0, 3.11]
```

Comment:
So 3.7 should drop out of this (unsupported), and the same comment goes for the first piece about needing the project to support newer Python versions.